README.md

```diff
@@ -72,6 +72,37 @@ asyncio.run(client.subscribe(query=query, handle=print))
 # ...
 ```
 
+## Advanced Usage
+
+### Disable SSL verification
+
+Set `options` to `{"verify": False}` ether when instantiating the `GraphqlClient` class.
```

Comment:
Since we're using `**kwargs` now, I think this can be reworded for clarity.

```suggestion
Set the keyword argument `verify=False` ether when instantiating the `GraphqlClient` class.
```